.ado/templates/build-rnw.yml

```diff
@@ -49,8 +48,9 @@ steps:
       msbuildArguments:
         /p:PreferredToolArchitecture=${{parameters.preferredToolArchitecture}}
         /p:PlatformToolset=${{parameters.platformToolset}}
-        ${{parameters.msbuildArguments}}
         /p:BaseIntDir=$(BaseIntDir)
+        /p:PATCH_RN=${{parameters.applyRnPatches}}
```

Comment:
Want to rename `PATCH_RN` in a followup change to use normal conventions for properties. This was originally following the lead of OSS_RN but shouldn't be like this.